kapitan/utils.py

```diff
@@ -111,7 +111,7 @@ def sha256_string(string):
     return sha256(string.encode("UTF-8")).hexdigest()
 
 
-def render_jinja2_file(name, context, jinja2_filter_paths=[]):
+def render_jinja2_file(name, context, jinja2_filters=os.path.join('lib', 'jinja2_filters.py')):
```

Comment:
Can you put `os.path.join('lib', 'jinja2_filters.py')` in a readonly var in `kapitan/inputs/jinja2.py`  and then reuse it everywhere instead of defining a hardcoded value in 2 places? Otherwise changing this in the future might be a headache.

Reply:
It works fine in cli.py but I think it is creating cyclic import problem between utils and jinja2 modules.

Reply:
@uberspot can you reproduce this behaviour ?

Reply:
If the cyclic dependency doesn't occure in jinja2_filters.py then you can put it there. It's mostly to avoid duplication in obscure places. And add a comment too detailing what that path is exactly. :+1:

Reply:
Updated !